### PR TITLE
Freemium: update marketplace product card label

### PR DESCRIPTION
**Pull request by raicem**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Change the label for Freemium products in the In-App Marketplace.

### How to test the changes in this Pull Request:

1. Checkout this branch
2. In the `product-card-footer.tsx`, set `product.freemium_type = 'primary';` [in line 19. ](https://github.com/woocommerce/woocommerce/blob/71ebf5932437becb72e4fb65e2014aaf038d5bf9/plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx#L18)
     - This eliminates the need for connecting to a local WooCommerce.com database. The product endpoint doesn't yet have any freemium products. 
3. Get a new build
4. Go to the **WooCommerce > Extensions > Browse** and see the new "Free plan available" label (you may need to scroll down a bit as most products have **Add to Store** button still)
    image
5. Please test around this issue

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [x] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [x] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx [hunk 1]
```diff
@@ -83,17 +107,10 @@ function ProductCardFooter( props: { product: Product } ) {
...
-							? __( 'Free download', 'woocommerce' )
-							: currencySymbol + product.price
-					}
+					{ getPriceLabel() }
 				</span>
 				<span className="woocommerce-marketplace__product-card__price-billing">
-					{ product.price === 0
-						? ''
-						: __( ' annually', 'woocommerce' ) }
+					{ getBillingText() }
```
Comment (kdevnel) [inline plugins/woocommerce-admin/client/marketplace/components/product-card/product-card-footer.tsx:113]: This is so much better!

**Review by kdevnel — APPROVED** [review]
Tests well and works as expected. Thank you!

Just note, there are no links to the changes you mention in step 4 for the cache values. I didn't need them, but thought to mention it. Also, I only realised after reading the code that testing with a free product set as primary wouldn't work. 

PS - you have some E2E test failures, might be worth running them again.

**Comment by raicem** [comment]
> Just note, there are no links to the changes you mention in step 4 for the cache values. I didn't need them, but thought to mention it.

Thank you @kdevnel, I have fixed in case it's needed in the future. 

I have rebased this on top of trunk and hopefully that'll resolve the e2e tests :crossed_fingers:

**Review by kdevnel — APPROVED** [review]
Looks good! Now it seems the auto-changelog adder is failing to run. You could always add the changelog manually.

**Comment by raicem** [comment]
> Looks good! Now it seems the auto-changelog adder is failing to run. You could always add the changelog manually.

Yeah, I'm struggling with that now. I don't want to add a changelog item, I think this change is not worth it. I wanted to do a patch buy I haven't picked the **Type**. Maybe that's why. Re-running it now.

**Pull request merged**